packages/overlay/test/index.ts

```diff
@@ -47,6 +46,7 @@ export const runOverlayTriggerTests = (type: string): void => {
                                     justify-content: center;
                                 }
                             </style>
+                            <input type="text" />
```

Comment:
is this deliberate?

Reply:
Yes, the test was giving issues getting into "focus visible", and this makes it easier to enforce that by allowing the test to `Tab` from the `input` to the test element.